cognite/neat/rules/models/rules.py

```diff
@@ -649,6 +649,12 @@ class Property(Resource):
     # Asset and Relationship at the same time
     cdf_resource_type: list[str] = Field(alias="Resource Type", default_factory=list)
 
+    # container specific things, only used for advance modeling or auto-filled by neat
+    container: Container | None = Field(alias="Container", default=None)
+    container_property: str | None = Field(alias="Container Property", default=None)
+    index: bool | None = Field(alias="Index", default=False)
+    constraints: str | None = Field(alias="Constraints", default=None, min_length=1)
```

Comment:
The reason why we need to have this here is that we need to have some way of expressing the physical storage of the data? Which thus have to be done on a model level.